src/Background/stat_estimators.jl

```diff
@@ -20,3 +20,20 @@ julia> estimate_background(Mean, data, dims=1)
 struct Mean <: BackgroundEstimator end
 
 estimate_background(::Mean, data; dims = :) = mean(data, dims = dims)
+
+"""
+    Mode <: BackgroundEstimator
+
+This estimator returns the mode of the input.
+
+# Example
+```jldoctest
+julia> data = ones(5, 5);
+
+julia> estimate_background(Mode, data)
+1.0
+```
+"""
+struct Mode <: BackgroundEstimator end
+
+estimate_background(::Mode, data; dims = :) = mode(data)
```

Comment:
Please add a `!!! note` admonition in the docstring describing that this doesn't allow for the `dims` keyword argument.